### DISCUSSION TITLE
About "RealSense F455"

### DISCUSSION THREAD

**Discussion by Vermilion016** [community/community#35164] [Other Feature Feedback, Questions, & Ideas] [answered]
I bought "RealSense F455" the other day, and I'm looking for a file called "resid_sdk_v0.18.0_win64.exe" to start it.
It should be in "https://github.com/IntelRealSense/RealSenseID/releases" (URL), but for some reason I can't find it on the site, could you tell me where it is?

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by Vermilion016** [reply]
I see. I'll try.
Thanks for your cooperation.
